.github/workflows/nodejs.yml

```diff
@@ -0,0 +1,29 @@
+name: CI
+
+on: [push]
+
+jobs:
+  test:
+    name: Test on node ${{ matrix.node_version }} and ${{ matrix.os }}
```

Comment:
The long name here will make the CI status's pretty hard to read.

![Screen Shot 2019-10-02 at 9 43 23 AM](https://user-images.githubusercontent.com/12637/66049270-1d3c6e00-e4f9-11e9-92e7-14ff4f83ae98.png)

If you make this `Tests` only, the CI status's _will_ still include the dynamic bits automatically.

```suggestion
    name: Tests
```